lib/wavefile/chunk_readers/smpl_chunk_reader.rb

```diff
@@ -0,0 +1,118 @@
+module WaveFile
+  module ChunkReaders
+    # Internal
+    class SmplChunkReader < BaseChunkReader    # :nodoc:
+      # Public: Returns the ID of the manufacturer
+      # See the list at https://www.midi.org/specifications-old/item/manufacturer-id-numbers
+      attr_reader :manufacturer_id
+
+      # Public: Returns the ID of the product
+      attr_reader :product_id
+
+      # Public: Returns the length of each sample in nanoseconds
+      attr_reader :sample_duration
+
+      # Public: Returns the MIDI note number of the sample (0-127)
+      attr_reader :unity_note
+
+      # Public: Returns the fraction of a semitone up from the specified MIDI unity note field.
+      # A value of 0x80000000 means 1/2 semitone (50 cents) and a value of 0x00000000 means no fine tuning between semitones.
+      # - https://sites.google.com/site/musicgapi/technical-documents/wav-file-format
+      # Integer
+      attr_reader :pitch_fraction
```

Comment:
Since it seems like the expected value is a number between 0 and 1, it might be better to convert the number to that range, instead of an integer between 0 and 4,294,967,295. I.e. if that value read from the file is `0x80000000`, then return `0.5` instead of `2_147_483_648`.